### PR TITLE
Topics sorted alphabetically

### PR DESCRIPTION
**Pull request by lazzy-panda**
I sorted topics tags alphabetically before render.

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [x] Unit checks
- [ ] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

File: kafka-ui-react-app/src/components/Connect/List/ListItem.tsx [hunk 1]
```diff
@@ -52,6 +52,10 @@ const ListItem: React.FC<ListItemProps> = ({
     setDeleteConnectorConfirmationVisible(false);
   }, [clusterName, connect, dispatch, name]);
 
+  const sortedTopics = topics?.length
+    ? [...topics].sort((a: string, b: string) => a.localeCompare(b))
+    : [];
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/List/ListItem.tsx:57]: It is a work of selector

File: kafka-ui-react-app/src/components/Connect/List/ListItem.tsx [hunk 2]
```diff
@@ -72,7 +76,7 @@ const ListItem: React.FC<ListItemProps> = ({
       <td>{connectorClass}</td>
       <td>
         <S.TagsWrapper>
-          {topics?.map((t) => (
+          {sortedTopics?.map((t: string) => (
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/List/ListItem.tsx:79]: sortedTopics is array. no need to use `?`

File: kafka-ui-react-app/src/components/Connect/List/__tests__/ListItem.spec.tsx [hunk 3]
```diff
@@ -52,6 +52,13 @@ describe('Connectors ListItem', () => {
     expect(screen.getAllByRole('cell')[6]).toHaveTextContent('2 of 2');
   });
 
+  it('topics tags are sorted', () => {
+    render(setupWrapper());
+    expect(screen.getAllByRole('link')[1]).toHaveTextContent('a');
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Connect/List/__tests__/ListItem.spec.tsx:57]: and here we can put it in a variable and reuse it for array , in order not to call it everytime

File: kafka-ui-react-app/src/components/Connect/List/ListItem.tsx (hunk 1 above)
Comment (lazzy-panda) [reply kafka-ui-react-app/src/components/Connect/List/ListItem.tsx]: moved to redux

File: kafka-ui-react-app/src/components/Connect/List/__tests__/ListItem.spec.tsx (hunk 3 above)
Comment (lazzy-panda) [reply kafka-ui-react-app/src/components/Connect/List/__tests__/ListItem.spec.tsx]: fixed

File: kafka-ui-react-app/src/redux/reducers/connect/selectors.ts [hunk 4]
```diff
@@ -52,6 +52,19 @@ export const getFailedConnectors = createSelector(
...
+export const getSortedTopics = createSelector(
+  connectState,
+  ({ connectors }) => {
+    return connectors.map((connector) => {
+      return connector.topics?.length
+        ? [...connector.topics].sort((a: string, b: string) =>
+            a.localeCompare(b)
+          )
+        : connector.topics;
+    });
```
Comment (workshur) [inline kafka-ui-react-app/src/redux/reducers/connect/selectors.ts:64]: return connector.topics?.sort((a, b) => a.localeCompare(b))

Comment (workshur) [reply kafka-ui-react-app/src/redux/reducers/connect/selectors.ts:64]: As far as I remember, localeCompare is a very slow operation and should only be used for strings containing non-latin characters.

Comment (lazzy-panda) [reply kafka-ui-react-app/src/redux/reducers/connect/selectors.ts:62]: fixed

Comment (workshur) [reply kafka-ui-react-app/src/redux/reducers/connect/selectors.ts:62]:
```diff
- return connector.topics?.length
-         ? [...connector.topics].sort()
-         : connector.topics;
+ return connector.topics?.sort((a, b) => a.localeCompare(b))
```

File: kafka-ui-react-app/src/components/Connect/List/ListItem.tsx [hunk 5]
```diff
@@ -72,7 +72,7 @@ const ListItem: React.FC<ListItemProps> = ({
       <td>{connectorClass}</td>
       <td>
         <S.TagsWrapper>
-          {topics?.map((t) => (
+          {topics?.map((t: string) => (
```
[the remark this answers is not in the document]
Comment (lazzy-panda) [reply kafka-ui-react-app/src/components/Connect/List/ListItem.tsx]: removed

File: kafka-ui-react-app/src/redux/reducers/connect/selectors.ts (hunk 4 above)
Comment (lazzy-panda) [reply kafka-ui-react-app/src/redux/reducers/connect/selectors.ts]: fixed

**Review by workshur — APPROVED** [review]

**Pull request merged**